hack/generate-manifest.sh

```diff
@@ -325,7 +325,8 @@ fi
 if [ "$MODE" == "dev" ]; then
     if [[ -z "$IMG_NAME" ]]; then
         if $COVERAGE; then
-            HELM_VALUES+=("image.repository=antrea/antrea-ubuntu-coverage")
+            HELM_VALUES+=("controllerImage.repository=antrea/antrea-controller-ubuntu-coverage" \
```

Comment:
Please check and update prepare-asserts.sh and generate-standard-manifests.sh for this change. 
e.g: https://github.com/antrea-io/antrea/blob/main/hack/generate-standard-manifests.sh#L115

Reply:
sure.